pom.xml

```
@@ -25,7 +25,7 @@
 
         <mule.http.connector.version>1.7.3</mule.http.connector.version>
         <muleSdkApiVersion>0.3.0</muleSdkApiVersion>
-        <soap.engine.version>1.8.1</soap.engine.version>
+        <soap.engine.version>1.9.0-SNAPSHOT</soap.engine.version>
```

We should use final versions only when releasing to prod 
```suggestion
        <soap.engine.version>1.8.6</soap.engine.version>
```